### ISSUE TITLE
Dead link in Other Important Info.md

### ISSUE THREAD

**Issue by Ding-Fan** [JordanMartinez/purescript-jordans-reference#487]
## Context

Section:
  Getting Started

Issue Type:
  Bug

Folder/File(s):
  - Getting Started/Other Important Info.md

## Body

```
- **Refer to [`dwhitney`'s fork of the book's code](https://github.com/dwhitney/purescript-book/tree/0.12)**, which is currently being updating to use `0.13.6`.
```

The link "https://github.com/dwhitney/purescript-book/tree/0.12" is dead.

**Comment by JordanMartinez** [comment]
Has this been fixed in the `v0.14-syntax` branch?

**Comment by JordanMartinez** [comment]
Thanks for investigating that. See [my comment](https://github.com/JordanMartinez/purescript-jordans-reference/issues/480#issuecomment-640234262) for why I haven't done much on this repo since February/March.
